kitsune/sumo/templates/base.html

```diff
@@ -237,10 +237,6 @@
 {% endif %}
 {# End Optimizely #}
 
-{% if include_geoip %}
```

Comment:
I think `include_geoip` is now unused, and can be removed. It is defined [here](https://github.com/mozilla/kitsune/blob/master/kitsune/sumo/context_processors.py#L23-L29).